next.config.js

```diff
@@ -3,12 +3,15 @@ const nextConfig = {
   experimental: {
     appDir: true,
     runtime: "experimental-edge",
```

Comment:
fyi this option is deprecated - you should put `export const runtime = "edge"` on individual pages instead

Reply:
This option still works and I really hate putting it on every page, I want edge to be default

Reply:
Lee disagrees https://twitter.com/leeerob/status/1638985828482400256?s=46&t=Uy_Hg1WWwsYDe1YleBLsaA ;)